### ISSUE TITLE
Replace tab-driven feature navigation with arrow keys

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#678]
For reference, [markers in Google Maps](https://developers.google.com/maps/documentation/javascript/examples/marker-accessibility) are navigated using arrow keys (though they fail to use appropriate roles for elements per the ARIA design patterns such that users can expect them to be navigated using arrow keys. I may file a bug when I have time to).

_Originally posted by @Malvoz in https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/535#issuecomment-945930374_

**Comment by prushforth** [comment]
I suggest to use a simple replacement of left-arrow and up-arrow => shift-tab, while down-arrow and right-arrow  => tab.

An open question, perhaps for @Malvoz, is how / if to provide audible feedback to screen reader users when the last feature  is reached. Similarly when shift-tabbing backwards, would be ideal to give audible feedback for when first feature of sequence is reached.

**Comment by ben-lu-uw** [comment]
At the moment the linked PR makes it so that the user has to tab in to the features, navigate the features using arrow keys and tab again if they want to reach the controls. Navigating through the features with only arrow keys would "trap them" at the last feature. 
Either that behavior could be removed somehow or some feedback to the user could be added as Peter mentioned.

**Comment by prushforth** [comment]
It's working pretty well. A distinctive sound would be good, but next best would be a localized announcement like "last feature, hit tab to proceed/ shift-tab to go back". ymmv.

**Comment by prushforth** [comment]
We're going to have to figure out how to integrate this with the feature index as the next tab stop.

**Comment by Malvoz** [comment]
> how / if to provide audible feedback to screen reader users when the last feature is reached. Similarly when shift-tabbing backwards, would be ideal to give audible feedback for when first feature of sequence is reached.

I think either:

For <kbd>up</kbd> and <kbd>left</kbd>: If focus is on the first feature, moves focus to the last feature.
For <kbd>right</kbd> and <kbd>down</kbd>: If focus is on the last feature, moves focus to the first feature.

similar to the [keyboard support for the APG menubar](https://www.w3.org/WAI/ARIA/apg/example-index/menubar/menubar-navigation.html#kbd1_label).

Or similar to [Grids](https://www.w3.org/WAI/ARIA/apg/patterns/grid/#keyboard-interaction-for-data-grids): don't move the focus.

---

One concern with implementing this: how would screen reader users know they can use arrow keys to navigate features? I think that's where some of the widget roles related to both (e.g.) menubars and grids come into play. But the patterns tend to expect certain roles on the focusable items where we'd perhaps want `button` or `link` for features. I think [focusgroups](https://open-ui.org/components/focusgroup.explainer/) can help in the future, and perhaps MapML can even specify that features ought to behave as focusgroup items. Text instructions (https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/267) can also be helpful, or even `aria-label` similar to the Google Maps site:

![map arrow keys aria-label](https://user-images.githubusercontent.com/26493779/182942172-a436b785-c610-401c-8587-5fdff44b92bb.png)

**Comment by prushforth** [comment]
> Or similar to [Grids](https://www.w3.org/WAI/ARIA/apg/patterns/grid/#keyboard-interaction-for-data-grids): don't move the focus.  

An observation from a quick look at that: page down and page up are used as "accelerators", moving up/down (by analogy forward/backward) some predefined number greater than 1 of table rows, and Ctrl+Home Ctrl+End move to the first/last rows.  Not to add work to this PR, but these might also be useful for features.  Similarly, for later / another PR.
